doc/releases/changelog-dev.md

```diff
@@ -305,6 +305,11 @@
 
 <h3>Breaking changes</h3>
 
+* The tape method `get_operation` can also now return the operation index in the tape, and it can be
```

Comment:
is this a breaking change because a warning is raised?

Reply:
Partly that and just that users are aware, that method `get_operator` will have a different signature after `0.30.0`

Reply:
kk. not sure i'd strictly call this "breaking", but putting it here is probably the best way to let users know asap so lgtm

Reply:
I agree, it's not really a breaking change. Maybe we could have a "Deprecations" section in the changelog? :thinking: @albi3ro is there a standard way to announce deprecations in the changelog?